3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,131 @@
+
+let yearOfBirth = 1982;
+let yearFuture = 2022;
+let age = yearFuture - yearOfBirth;
+console.log(`you will be ${age} years old in ${yearFuture}`);
+
+//Dog year of birth
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let humanYears = dogYearFuture - dogYearOfBirth; 
+let dogYear = humanYears*7; 
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+let dogAgeInDogYear = dogYear;
+
+console.log(`Your dog will be ${dogYear} dog years in ${dogYearFuture}`);
+}
+else {
+    console.log(`Your dog will be ${humanYears} human years old in ${dogYearFuture}`);
+}
+
+//House price generator for Your friend Peter is considering a house that is 8m wide, 
+//10m deep and 10m high. The garden size is 100m2. The house costs 2.500.000. 
+//Your friend Julia is considering a house that is 5m wide, 11m deep and 8m high. 
+//The garden size is 70m2. This house costs 1.000.000.
+//Formula is housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+let houseDetails = [
```

Comment:
cool 😎 that you used an array here; 

better to use const if you are not planing to change original array